bin/carrot-server.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
 START_HOME=$PWD
+# within instruction $(dirname "$(realpath "$0")") you can start server from any directory. No just from bin
```

Comment:
realpath command does not exist in Mac OS

Reply:
I think it doesn't exists by default but could be installed. However I didn't find another common solution for Mac and Linux.
Not critical at this time but if we want to use auto-start script it does. 
https://ports.macports.org/port/realpath/